docs/UserGuide.adoc

```diff
@@ -241,25 +253,36 @@ Format: `classify RATING`
 *RATING* {‘easy’ | ‘good’ | ‘hard’ | ‘review’}
 ****
 
-=== Sort current list in lexicographical order : `sort`
-If user is currently in a deck, sort all cards in alphabetical order according to the question. +
-If user is not in a deck, sort all decks in alphabetical order according to the deck's name. +
+// tag::sort[]
+=== Sort : `sort`
+Sort the current list in lexicographical order.
+
+* If user is currently inside a deck, sort all cards according to their question. +
+* If user is not in a deck, sort all decks according to the their name. +
```

Comment:
Looks good! Specifiying the exact conditions under which our command works will help decrease the amount of bugs that will be found tomorrow